connectivity/drivers/cellular/Samsung/TARGET_S5JS100/TOOLCHAIN_ARM_STD/LICENSE

```diff
@@ -0,0 +1,18 @@
+/****************************************************************************
```

Comment:
Can you add readme to TARGET_S5JS100 folder, where are all these prebuilt libraries located - it would be good to documented what toolchain versions were used to generate these libraries and what this library actually is about - from what I understand it provides a cellular driver for a specific cellular module that is on board or ?

Reply:
OK, we will add readme.